lib/heroku/kensa/check.rb

```diff
@@ -173,6 +173,14 @@ def call!
           response.is_a?(Hash) && response.has_key?("id")
         end
 
+        check "id does not contain heroku_id" do
+          if response["id"] && response["id"].respond_to?(:include?) && response["id"].include?(data["heroku_id"])
```

Comment:
re: `response["id"]` & string vs int – I was mostly going off of the other tests. There's some that use ints and some that return a `{ id: nil }` which the check above this one would not catch.
re: heroku_id – I took "heroku id" in the TP card to mean the hid.

I'll update everything to match what what you've mentioned. Thanks!